brd-ios/breadwallet/SimpleTextField.swift

```diff
@@ -105,6 +127,21 @@ class SimpleTextField: UIView, UITextFieldDelegate {
         }
     }
     
+    var isValid: Bool {
```

Comment:
validation of textField content should be done through a VIP cycle

Reply:
or via a viewModel validation closure

Reply:
already fixed. will push.